docs/keptn_annotations.md

```diff
@@ -6,7 +6,7 @@ You can provide some extra information about your Keptn integrations by using th
 
 - **keptn/kind** *(string)*
 
-This annotation allows providing the kind of the integration. Supported values: *service* and *sli-provider*. It's possible to add multiple kinds by using a comma separated list, like shown in the example below.
+This annotation allows providing the kind of the integration. Supported values: *webhook*, *testing*, *deployment*, *notification*, *observability* and *sli-provider*. It's possible to add multiple kinds by using a comma separated list, like shown in the example below.
```

Comment:
Can we please extend the list by the value: `remediation` in order to cover the auto-remedation use-cases.